### PR TITLE
Fix dead links from side bar

### PR DESCRIPTION
**Pull request by nicolasstucki**

### CODE REVIEW

File: docs/sidebar.yml [hunk 1]
```diff
@@ -11,8 +11,6 @@ sidebar:
           url: docs/usage/cbt-projects.html
         - title: Dottydoc
           url: docs/usage/dottydoc.html
-        - title: Migrating
-          url: docs/usage/migrating.html
```
Comment (DarkDimius) [inline docs/sidebar.yml]: but there is a `./docs/usage/migrating.md`

Comment (nicolasstucki) [inline docs/sidebar.yml]: Not on master

Comment (nicolasstucki) [inline docs/sidebar.yml]: Removed in ddd48da

Comment (DarkDimius) [inline docs/sidebar.yml]: you're right.

Comment (nicolasstucki) [inline docs/sidebar.yml]: @DarkDimius so we should merge?

**Pull request merged**